testsuite/src/it/java/org/graalvm/tests/integration/utils/Logs.java

```diff
@@ -99,7 +99,7 @@ public static void checkThreshold(Apps app, Mode mode, long executableSizeKb, lo
                     "Skipping checking thresholds.");
             return;
         }
-        final String propPrefix = (IS_THIS_WINDOWS ? "windows" : "linux") + ((mode != Mode.NONE) ? "." + mode : "");
+        final String propPrefix = (IS_THIS_WINDOWS ? "windows" : "linflight-native.jfrux") + ((mode != Mode.NONE) ? "." + mode : "");
```

Comment:
Should the `threshold.properties` property names also be updated?

Reply:
I will take a look.

`linflight-native.jfrux` being lin**flight-native.jfr**ux jammed inside the word `linux` seems like some kind of typo...